### DISCUSSION TITLE
Editing comments in Python is not as same as in Lua.

### DISCUSSION THREAD

**Discussion by otakutyrant** [neovim/neovim#24237] [Q&A] [answered]
That said, in Lua files, when I press return key in a comment line, Neovim will automatically begin the new line with comments `--`. When there are two comments and if I press `J` in the first one, Neovim will automatically combine two comment lines into a comment line without interpolating `--` in the mid. Sometimes it is convenient.

But these two behaviours do not happen in editing comments in Python files. Is there any related configuration?

**Comment by clason** [comment, marked as answer]
`:h 'formatoptions'` (set by `ftplugin/lua.vim` but not by `ftplugin/python.vim`)

**Reply by otakutyrant** [reply]
SUPER! Thank you.
